.github/workflows/publish.yml

```diff
@@ -4,7 +4,6 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - lab
```

Comment:
Will `lab` be gone after this? Or is this temporary?

edit: ah you mentioned as well that this workflow will go away as a whole

Reply:
Yes I moved `lab` to deploy as it already handles all the bits for CloudFront and Lambda Edge for redirects.